includes/User.php

```diff
@@ -822,37 +821,17 @@ public function isValidPassword( $password ) {
 	 * @return mixed: true on success, string or array of error message on failure
```

Comment:
Is it still true? I can clearly see it can return false. Also not sure if it returns an array anywhere.
If the callers don't expect false as a result, we can always return 'unknown error' string or something similar.